PyRDF/Node.py

```diff
@@ -194,4 +194,13 @@ def graph_prune(self):
             
             return False
 
+        elif not self.children and self.value:
+
+            ### If the current node's value was already
+            ### computed, it should get pruned only if it's
+            ### an action node.
+
+            if self.operation and self.operation.is_action():
```

Comment:
Isn't this condition producing doing the contrary than what one would expect? Here you are **not** pruning if it is an action that already has a value, and that is precisely what you **do** want to prune, right?